symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/form/FormReplyContext.java

```diff
@@ -35,6 +35,7 @@ public FormReplyContext(V4Initiator initiator, V4SymphonyElementsAction eventSou
 
   @API(status = API.Status.EXPERIMENTAL)
   public String getFormValue(String fieldName) {
```

Comment:
should we add a @Nullable on the method? (or consider using optional here? as the api is experimental we can still change it)

Reply:
+1 

We could also replace `API.Status.EXPERIMENTAL` by `API.Status.STABLE` and document this method with Javadoc

Reply:
done